src/index.css

```diff
@@ -16,6 +16,10 @@ textarea {
   padding-right: 0.75ch; /* TODO: Future task: Override this in MuiTheme */
 }
 
+a {
+  text-decoration: none;
```

Comment:
Are we sure, we want to remove all `a` tags text decoration? (instead of removing it only at a local level)  

We might have hyper-links in the future

Reply:
When is the last time you saw hyper link in a website? nowdays it's all buttons